test/pred-sets/ref/ccb_quad.inv

```diff
@@ -24,128 +24,174 @@ weighted_labeled_examples 8
 weighted_labels 0
 weighted_unlabeled_examples 0
 example_number 8
-total_features 758
+total_features 1151
 total_weight 15.2712
```

Comment:
is this calculating more interactions than before?

Reply:
Yes, CCB will interact more namespaces than before. The old code didn't add all the intended interactions. 

The set of interactions for CCB (see https://github.com/VowpalWabbit/vowpal_wabbit/pull/2993/files#diff-bf3dd006bfbdeca21413c272e2a5c433c01659ccd060d2423df5c9c25ca877c8) is:
1. For each existing in interaction insert two new interactions:
    1. The existing one plus ccb_id_namespace
    2. The existing one plus ccb_slot_namespace
2. wildcard +  ccb_id_namespace
3. wildcard +  ccb_slot_namespace

Reply:
Will this change cause difference in behaviour in existing models?

Reply:
are we sure those were intended to be added in the wildcard interactions and not only as an extra interaction at then end of the existing ones (i.e. just case `1.` described above)?